app/views/global/_social_sharing.html.erb

```diff
@@ -14,14 +14,14 @@
   </a>
 
   <a 
-    href="https://twitter.com/intent/tweet/?text=<%= url_encode('Check out the Collated Coral website') %>&url=<%= encoded_page_url %>" 
+    href="https://twitter.com/intent/tweet/?text=<%= url_encode('Coral Reef Funding Landscape - Explore international funding allocations for the sustainable management of coral reefs and related mangroves and seagrass ecosystems.') %>&url=<%= encoded_page_url %>" 
     title="Share Collated Coral on Twitter" 
     class="social--share social--twitter"
     target="_blank">
   </a>
 
   <a 
-    href="mailto:?subject=<%= url_encode('Collated Coral website') %>&body=<%= encoded_page_url %>" 
+    href="mailto:?subject=<%= url_encode('Coral Reef Funding Landscape') %>&body=<%= url_encode('Coral Reef Funding Landscape') %>%0D%0A%0D%0A<%= url_encode('Search the database to explore international funding allocations for the sustainable management of coral reefs and related mangroves and seagrass ecosystems.') %>%0D%0A%0D%0A<%= encoded_page_url %>"
```

Comment:
Can we please use helpers for these?
Params are quite complex so it would better to have those in variables for example, so the href looks cleaner.